### ISSUE TITLE
Add a way to store extra data (JSON) to User profiles

### ISSUE THREAD

**Issue by scott** [helpyio/helpy#521]

**Comment by lccezinha** [comment]
@scott what about it ? http://nandovieira.com/using-postgresql-and-jsonb-with-ruby-on-rails

**Comment by scott** [comment]
Yeah I was thinking of this or just serializing into a text column.

**Comment by lccezinha** [comment]
I'll open a PR with this enhancement ASAP ok ?

**Comment by scott** [comment]
That would be awesome.  I picture using the User#create API as the primary way of adding json data to the User.  Can you also add this?
